config/samples/enclave-cc/base/ccruntime-enclave-cc.yaml

```diff
@@ -77,23 +77,23 @@ spec:
             type: ""
           name: etc-systemd-system
         - hostPath:
-            path: /var/run/dbus
+            path: /var/run/dbus/system_bus_socket
             type: ""
           name: dbus
         - hostPath:
-            path: /run/systemd
+            path: /run/systemd/system
             type: ""
           name: systemd
     preInstall:
-      image: quay.io/confidential-containers/container-engine-for-cc-payload:1def978fdf6b0cb2383d43970c3dc484df3cad54
+      image: quay.io/confidential-containers/container-engine-for-cc-payload:98a790e8abdcc06c4b629b290ebaa217bf82e305
```

Comment:
There is a new container engine image as well that is 2 days old rather than 17

Reply:
The new container image was the one I used for testing the `chroot`, which didn't work.

Reply:
Cool - we just want to be aware that latest is pointing at it (in the unlikely event that anyone is using it)